torch/testing/_internal/common_utils.py

```diff
@@ -1044,7 +1044,7 @@ def _compareScalars(self, a, b, *,
                 rtol, atol = 0, 0
         rtol = cast(float, rtol)
         atol = cast(float, atol)
-        atol = max(atol, self.precision)
+        atol = max(atol, self.precision) if atol is not None else self.precision
```

Comment:
`atol` cannot be `None` here anymore, so `assert atol is not None` on the line right above this one seems like the clearer fix.